sklearn/preprocessing/tests/test_data.py

```diff
@@ -2224,3 +2225,218 @@ def test_quantile_transform_valid_axis():
 
     assert_raises_regex(ValueError, "axis should be either equal to 0 or 1"
                         ". Got axis=2", quantile_transform, X.T, axis=2)
+
+
+def _generate_random_features_matrix(n_values=3, size=10):
+    rng = np.random.RandomState(0)
+    X = rng.randint(n_values, size=(size, n_values))
```

Comment:
Is there a reason why `n_features == n_values` ?
Using `n_samples` and `n_features` variable names would be clearer.